### PR TITLE
[WIP] Image Component (MDX)

### PR DESCRIPTION
**Pull request by imadarai**
- [x] Pull Main into Develop 
- [x] Image Component 
- [x] Add Image Component to Layout through MDX Provider 
- [x] Update All image tags in Documentation 
- [x] QA `develop` mode 
- [ ] QA `production` mode in staging (Awaiting PR Approvals)

### CODE REVIEW

**Comment by imadarai** [comment]
@sacha-l Can you please review the markdown files and all the image tag updates?  
It looks like the files were already moved to `src/images/docs` & `src/images/tutorials`.  So I'm hoping to maintain that convention, if thats what the doc team wants.  
Also would it be ok to remove the doc related images from `static/img` now? Everything that I updated now is being pulled from `src/images`, so it should be good to delete.

**Comment by NukeManDan** [comment]
I already did this in https://github.com/substrate-developer-hub/substrate-docs/pull/530

you can `git  cherry-pick` the commit from the main branch (squashed) or from the branch something more specific

File: v3/tutorials/11-kitties-workshop/b-kitties-frontend/index.mdx [hunk 1]
```diff
@@ -416,7 +416,7 @@ Our `KittyCards.js` component will have three sections to it:
 
   See the screenshot below for reference:
 
-   ![kitty-transfer-shot.png](../../../../src/images/tutorials/11-kitties-workshop/kitty-transfer-shot.png)
+   ![kitty-transfer-shot.png](kitty-transfer-shot.png)
```
Comment (NukeManDan) [inline v3/tutorials/11-kitties-workshop/b-kitties-frontend/index.mdx:419]: so the convention you want is no hierarchy? it just discovers where an image us so long as it exists in `src/images`?

Comment (imadarai) [reply v3/tutorials/11-kitties-workshop/b-kitties-frontend/index.mdx:419]: I am just working with the requirement that was given to me.  As, in markdown, only provide the name of the file, not the tree.  
If we want to update this further, perhaps open it up for a discussion with the rest of the team.  Come up with either 
- Naming Convention, OR 
- Folder/Naming Convention

Let me know!

**Comment by hugurp** [comment]
There might be the possibility of conflicting file names.

It seems that if files have the same name, but are in different folders (`docs`, `tutorials`), since there is no folder reference, they would be sourced by gatsby with the same name.

**Comment by NukeManDan** [comment]
> There might be the possibility of conflicting file names.

true, but not likely. And this would cause error that might be helpful so we don't _duplicate_ images. We pull from the _same dir for all of them, so there is no ambiguity

File: v3/tutorials/11-kitties-workshop/b-kitties-frontend/index.mdx (hunk 1 above)
Comment (rmnprkrl) [reply v3/tutorials/11-kitties-workshop/b-kitties-frontend/index.mdx:419]: As we will be migrating to new infrastructure based on `mkdocs` if we want to do this before in this stack already we should go with the planned structure schema.

@imadarai Should be moved to `/media/images/<section>/<category>` to mirror the content structure.
Eg.: `/media/images/docs/02-concepts/:image.png`

Depends on you @sacha-l @NukeManDan if you want to go through this already now, as you will be also changing the categories etc.. (maybe we just do only one level less before the migration `/media/images/<section>/` and add category sub-folders in new migration?)

Comment (NukeManDan) [reply v3/tutorials/11-kitties-workshop/b-kitties-frontend/index.mdx:419]: @lisa-parity should make the call on the structure she wants.
IMHO a verbose and complete internal dir path is better as it's more clear to everyone (including outside contributors) where things live, it's not "magicly" found by our rendering engine.

**Comment by rmnprkrl** [comment]
@imadarai it is being addressed here: https://github.com/substrate-developer-hub/substrate-docs/pull/732
I would suggest close this PR

**Comment by imadarai** [comment]
Closing PR as changes are being handled in #732

**Pull request closed without merging**